frontend/server/src/DAO/Runs.php

```diff
@@ -244,6 +246,66 @@ final public static function getAllRuns(
                 $where[] = 's.verdict = ?';
                 $val[] = $verdict;
             }
+        } else {
+            if (!is_null($execution)) {
+                if ($execution === 'EXECUTION_INTERRUPTED') {
+                    $where[] = 's.verdict IN(?, ?, ?, ?, ?, ?)';
+                    $val[] = 'ML';
+                    $val[] = 'MLE';
+                    $val[] = 'TLE';
+                    $val[] = 'OLE';
+                    $val[] = 'TO';
+                    $val[] = 'OL';
+                } elseif ($execution === 'EXECUTION_RUNTIME_ERROR') {
+                    $where[] = 's.verdict IN(?, ?)';
+                    $val[] = 'RE';
+                    $val[] = 'RTE';
+                } elseif ($execution === 'EXECUTION_RUNTIME_FUNCTION_ERROR') {
+                    $where[] = 's.verdict IN(?, ?)';
+                    $val[] = 'OF';
+                    $val[] = 'RFE';
+                } elseif ($execution === 'EXECUTION_COMPILATION_ERROR') {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'CE';
+                } elseif ($execution === 'EXECUTION_VALIDATOR_ERROR') {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'VE';
+                } elseif ($execution === 'EXECUTION_JUDGE_ERROR') {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'JE';
+                } else {
+                    $where[] = 's.verdict IN(?, ?, ?)';
+                    $val[] = 'AC';
+                    $val[] = 'WA';
+                    $val[] = 'PA';
+                }
+            }
+
+            if (!is_null($output)) {
+                if ($output === 'OUTPUT_INTERRUPTED') {
+                    $where[] = 's.verdict IN(?, ?, ?, ?, ?, ?, ?, ?, ?)';
+                    $val[] = 'JE';
+                    $val[] = 'VE';
+                    $val[] = 'CE';
+                    $val[] = 'FO';
+                    $val[] = 'RFE';
+                    $val[] = 'RE';
+                    $val[] = 'RTE';
+                    $val[] = 'MLE';
+                    $val[] = 'TLE';
+                } elseif ($output === 'OUTPUT_INCORRECT') {
+                    $where[] = 's.verdict IN(?, ?)';
+                    $val[] = 'WA';
+                    $val[] = 'PA';
+                } elseif ($output === 'OUTPUT_EXCEEDED') {
+                    $where[] = 's.verdict IN(?, ?)';
+                    $val[] = 'OLE';
+                    $val[] = 'OL';
+                } else {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'AC';
+                }
```

Comment:
También para el output puedes usar la misma simplificación
```suggestion
                $outputArgs = \OmegaUp\Controllers\Run::OUTPUT[$output];
                $placeholders = array_fill(0, count($outputArgs), '?');
                $placeholders = join(',', $placeholders);

                $where[] = "s.verdict IN ({$placeholders})";
                $val = array_merge($val, $outputArgs);
```